frontend/src/app/app.component.spec.ts

```diff
@@ -16,16 +33,9 @@ describe('AppComponent', () => {
     expect(app).toBeTruthy();
   });
 
-  it(`should have as title 'frontend'`, () => {
+  it(`should have the title 'frontend'`, () => {
     const fixture = TestBed.createComponent(AppComponent);
     const app = fixture.componentInstance;
     expect(app.title).toEqual('frontend');
   });
-
-  it('should render title', () => {
```

Comment:
question: why was this test removed?

Reply:
Hi Eric, the test was originally generated by the Angular CLI. We removed this test since its test subject (the HTML element containing certain title) is no longer exist now. Currently, for similar test cases, we use another test (at line 36) to verify the title.